### PR TITLE
Policy/cleanup/testing

### PR DESCRIPTION
**Pull request by catlynbowles**
** Describe the functionality to be tested: **

(Bullet list feature/functionality)
- Add mobile responsive to policies page.
- Update styling to legend. 
- Add loading and error logic. 

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

**Review by Rallen13 — APPROVED** [review]
Looks great!

File: src/Components/PolicyCard/PolicyCard.scss [hunk 1]
```diff
@@ -13,10 +13,20 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    width: 30%;
+    width: auto;
+    text-align: center;
+  }
+}
+
+@media (min-width: 768px) {
```
Comment (lkarpins) [inline src/Components/PolicyCard/PolicyCard.scss:21]: Woooo!!! 🎉

**Review by lkarpins — APPROVED** [review]
Cat! It's a thing of beauty! Nice work, QUEEEEN.

File: src/Views/StatePoliciesView/StatePoliciesView.js [hunk 2]
```diff
@@ -74,43 +80,48 @@ const StatePoliciesView = () => {
...
+              filteredResults.filter((res) => legalResults.includes(res))
+            )
+          ) : searchInput ? (
+            generatePolicyCards(filteredResults)
+          ) : legalResults ? (
+            generatePolicyCards(legalResults)
+          ) : states ? (
+            generatePolicyCards(states)
+          ) : (
+            <h3>Loading...</h3>
```
Comment (irmakerdem) [inline src/Views/StatePoliciesView/StatePoliciesView.js:123]: We will change this to the Loading component ⏳

File: src/Components/PolicyCard/PolicyCard.scss (hunk 1 above)
Comment (irmakerdem) [inline src/Components/PolicyCard/PolicyCard.scss:21]: Responsive design! 👩‍🎨

File: src/Components/PolicyCard/PolicyCard.js [hunk 3]
```diff
@@ -4,16 +4,16 @@ import './PolicyCard.scss'
...
-    legal === '2' ? '#DF6349' : '#768CA0' 
+    return legal === 'at risk' ? '#D27736' : 
+    legal === 'illegal' ? '#DF6349' : '#768CA0' 
   }
 
   return (
-    <article className='info-strip' style={{border: '.1rem solid' + setColor()}}> 
-        <div className='name-container' style={{backgroundColor: setColor()}}>
+    <article data-cy='info-strip' className='info-strip' style={{border: '.1rem solid' + setColor()}}> 
+        <div tabIndex='0' data-cy='name-container' className='name-container' style={{backgroundColor: setColor()}}>
```
Comment (irmakerdem) [inline src/Components/PolicyCard/PolicyCard.js:13]: ♿️

**Review by irmakerdem — APPROVED** [review]
Hooray!! 🤡

**Pull request merged**